src/query/src/sql.c

```diff
@@ -22,9 +24,8 @@
 ** The following is the concatenation of all %include directives from the
 ** input grammar file:
 */
-#include <stdio.h>
-#include <assert.h>
 /************ Begin %include sections from the grammar ************************/
+#line 23 "sql.y"
```

Comment:
use lemon -l to remove location information from generated file?

Reply:
ok, I will see it the next time.

Reply:
ok. I will generate sql.c again